components/nvs_flash/include/nvs.h

```diff
@@ -661,14 +661,52 @@ esp_err_t nvs_entry_find(const char *part_name,
         nvs_type_t type,
         nvs_iterator_t *output_iterator);
 
+/**
+ * @brief       Create an iterator to enumerate NVS entries based on a handle and type
+ *
+ * \code{c}
+ * // Example of listing all the key-value pairs of any type under specified handle (which defines a partition and namespace)
+ *  nvs_iterator_t it = NULL;
+ *  esp_err_t res = nvs_entry_find_in_handle(<nvs_handle>, NVS_TYPE_ANY, &it);
+ *  while(res == ESP_OK) {
+ *      nvs_entry_info_t info;
+ *      nvs_entry_info(it, &info); // Can omit error check if parameters are guaranteed to be non-NULL
+ *      printf("key '%s', type '%d' \n", info.key, info.type);
+ *      res = nvs_entry_next(&it);
+ *  }
+ *  nvs_release_iterator(it);
+ * \endcode
+ *
+ * @param[in]   handle          Handle obtained from nvs_open function.
```

Comment:
Over here Doxygen complains that the argument name (handle) doesn't match the argument name in function declaration (c_handle). I'll fix this to use `handle` name in the declaration, consistent with the rest of the functions.

Reply:
Ah, whoops, ya. I originally wrote this change for the esp-idf 4.x and then ported it to 5.x. Seems I didn't fully fix the doxygen comments after that.